clients/privacy-center/app/server-environment.ts

```diff
@@ -31,7 +31,10 @@ import {
 
 export type PrivacyCenterServerSettings = Pick<
   PrivacyCenterSettings,
-  "SERVER_SIDE_FIDES_API_URL" | "FIDES_JS_MAX_AGE_SECONDS"
+  | "SERVER_SIDE_FIDES_API_URL"
+  | "FIDES_JS_MAX_AGE_SECONDS"
+  | "MISSING_EXPERIENCE_BEHAVIOR"
+  | "LOG_LEVEL"
```

Comment:
nit: There's some `console.debug`, `console.warn`, etc. usage in here that should swap over to the real logger, for consistency 👍

Reply:
Oof, realizing this ask would create a circular dependency between server environment and logger modules.